projects/plugins/jetpack/to-test.md

```diff
@@ -1,56 +1,96 @@
-## Jetpack 12.0
+## Jetpack 12.1
 
 ### Before you start:
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis) and see if there are any errors reported by Jetpack there.**
 - Use the "Debug Bar" or "Query Monitor" WordPress plugins to help make PHP notices and warnings more noticeable and report anything of note you see.
 
-### Blocks
+### Visual refresh
+Jetpack 12.1 brings a visual refresh to multiple pages of the admin area. Please visit them in your testing and make sure nothing looks off.
 
-Some blocks were moved from beta status to production and are now available to all users.
-No new features were added, so the only thing to test here is that they work as expected and show up without the beta blocks being enabled:
+#### Jetpack Debugger:
+- Connect Jetpack, scroll down to the bottom menu and click "Debug" (/wp-admin/admin.php?page=jetpack-debugger, not to be confused with the "Jetpack Debug" plugin).
+- Confirm the Debug page looks according to the design.
+- Optionally use "Jetpack Debug -> Broken Token" to break the blog token.
```

Comment:
> Optionally use "Jetpack Debug -> Broken Token" to break the blog token.

This step would be a bit confusing if someone hasn't used Jetpack Debug before.

Reply:
Removed that, makes sense, thanks!